pootle/apps/pootle_app/assets.py

```diff
@@ -50,6 +54,19 @@
 register('js_editor', js_editor)
 
 
+rel_path = os.path.join('js', 'select2_l10n')
+select2_l10n_dir = os.path.join(settings.WORKING_DIR, 'static', rel_path)
```

Comment:
Just curious, if we are already using this in static/ dir.  Why can't we just pick these out of node_modules dir in static dir?